src/prebid.js

```diff
@@ -251,8 +251,11 @@ $$PREBID_GLOBAL$$.setTargetingForAst = function() {
  * Returns a bool if all the bids have returned or timed out
  * @alias module:$$PREBID_GLOBAL$$.allBidsAvailable
  * @return {bool} all bids available
+ *
+ * @deprecated This function will be removed in Prebid 1.0
```

Comment:
So... if we assume that a user is seeing this comment, then the experience they most likely had is:

1. Hey... what are these warnings in my console?
2. Seeks more info. Finds their way to this comment.
3. But then how will I update my code to work with 1.0?
4. *throws tantrum*

If we want to avoid (4)s, then the comments here have to provide answers to (3). I don't have a magic bullet for this... but my best advice from past experience would be:

1. If there's some simple replacement method, list it here.
2. If not, mention the #github-issue-or-PR with conversation about the deprecation. If their use-case is captured there, they'll find an answer. If not, they can bring up their use-case so that we know about it.

You're doing this for a few methods... but I would definitely recommend doing it in all of them.